buildscripts/checkstyle.xml

```diff
@@ -376,15 +376,15 @@
     <module name="SuppressWarningsHolder"/>
     <!-- this enables CHECKSTYLE:OFF and CHECKSTYLE:ON comments -->
     <module name="SuppressionCommentFilter"/>
-    <module name="Regexp">
-      <!-- see comment in otel.errorprone-conventions.gradle.kts for why we disable the SystemOut
-           rule there and rely on this check -->
-      <property name="id" value="SystemOut"/>
-      <!-- . matches any character, so we need to escape it and use \. to match dots. -->
-      <property name="format"
-        value="(System\.out\.print|System\.err\.print|\.printStackTrace\(\)|Thread\.dumpStack)"/>
-      <property name="illegalPattern" value="true"/>
-    </module>
+<!--    <module name="Regexp">-->
```

Comment:
Why was this commented out?

Reply:
we need to allow prints until we will figure out tests